server/src/events.mjs

```diff
@@ -295,6 +298,9 @@ function emitResponse(finalResult, msg, userId, method) {
   //Update the call with event response
   updateCallWithResponse(method, eventMessage, "events", userId);
   wsArr.forEach((ws) => {
+    const userData = userManagement.getWsForUser(config?.dotConfig?.interactionService?.user);
+    createInteractionLog(eventMessage, method, null, userData); // creating interaction log and send it to the client
```

Comment:
this condition is placed inside `createInteractionLog` the function, so instead of having it in all the places where it is being called, I thought to put it inside the function itself